lib/global-styles.php

```diff
@@ -168,7 +170,13 @@ function gutenberg_experimental_global_styles_register_user_cpt() {
 }
 
 add_action( 'init', 'gutenberg_experimental_global_styles_register_user_cpt' );
-add_filter( 'block_editor_settings', 'gutenberg_experimental_global_styles_settings', PHP_INT_MAX );
+// This can be removed when plugin support requires WordPress 5.8.0+.
+if ( function_exists( 'get_block_editor_settings' ) ) {
```

Comment:
Hey @gziolo 👋 I was wondering if this should be `gutenberg_get_block_editor_settings` instead? and also in other places where it checks for `get_block_editor_settings` 

Isn't [this the function](https://github.com/WordPress/gutenberg/pull/30245/files#diff-3eacf766691cf7a47bc92c560709d919d19d247a1cf6cb6075a6862795bfd2c0R259) that should be checking for?

Just to double check because I'm getting some warnings while testing the block editor settings API endpoint 😅

Reply:
> Hey @gziolo 👋 I was wondering if this should be `gutenberg_get_block_editor_settings` instead? and also in other places where it checks for `get_block_editor_settings`

Can you share the warnings you get? This one is correct, we are testing against WordPress core. Whether it has `get_block_editor_settings` defined which means that it works with the new hook.

Reply:
I get the warnings when passing `post-editor` as a context. 

```
Deprecated: block_categories is <strong>deprecated</strong> since version 5.8.0! Use block_categories_all instead. in /var/www/html/wp-includes/functions.php on line 5242

Deprecated: block_editor_settings is <strong>deprecated</strong> since version 5.8.0! Use block_editor_settings_all instead. in /var/www/html/wp-includes/functions.php on line 5242

Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/wp-includes/functions.php: 5242) in /var/www/html/wp-includes/rest-api/class-wp-rest-server.php on line 164
```

Reply:
Do you have any plugins installed? Jetpack maybe?

Reply:
I did a clean install and it's working 🎉 , sorry for the noise! Thanks for the help 🙇

Reply:
Cool, it's always to double-check so I'm glad you brought it up 👍🏻